platform/ios/src/MGLMapboxEvents.m

```diff
@@ -122,21 +124,31 @@ - (void)updateDisablingConfigurationValuesWithNotification:(NSNotification *)not
 }
 
 + (void)setupWithAccessToken:(NSString *)accessToken {
-    NSString *semanticVersion = [NSBundle mgl_frameworkInfoDictionary][@"MGLSemanticVersionString"];
-    NSString *shortVersion = [NSBundle mgl_frameworkInfoDictionary][@"CFBundleShortVersionString"];
-    NSString *sdkVersion = semanticVersion ?: shortVersion;
+    int delayTime = 0;
```

Comment:
If possible, use the more specific `dispatch_time_t` type here.

Reply:
Oops, I thought I deleted this comment — it’s incorrect. The more specific type would be `uint_64`, but using `int` probably isn’t the end of the world.